lib/install/install.go

```diff
@@ -403,6 +406,9 @@ func (i *Installer) Wait() error {
 			return nil
 		case event := <-i.EventsC:
 			if event.Error != nil {
+				if utils.IsContextCancelledError(event.Error) {
```

Comment:
Ignore the `context is closing` errors as spam.

Reply:
Should this be suppressed here, or should it be up to the caller on whether to suppress logging of a context cancellation?